cloudify_cli/commands/deployments.py

```diff
@@ -727,3 +730,34 @@ def summary(target_field, sub_field, logger, client, tenant_name,
         items,
         'Deployment summary by {field}'.format(field=target_field),
     )
+
+
+@cfy.command(name='set-site',
+             short_help="Set the deployment's site [manager only]")
+@cfy.argument('deployment-id')
+@cfy.options.site_name
+@cfy.options.detach_site
+@cfy.options.common_options
+@cfy.assert_manager_active()
+@cfy.pass_client(use_tenant_in_header=True)
+@cfy.pass_logger
+def manager_set_site(deployment_id, site_name, detach_site, client, logger):
+    """Set the deployment's site
+
+    `DEPLOYMENT_ID` is the id of the deployment to update
+    """
+    if not (site_name or detach_site):
```

Comment:
this can be done by the mutually_exclusive feature of click

Reply:
They have mutually_exclusive, this is for the case that there is no `site_name` and `detach_site` is False.